### PR TITLE
Add warning, file download and numbered steps icons that were originally fetched from govuk_frontend_toolkit

### PR DESCRIPTION
**Pull request by vanitabarrett**
## What
Adds back icons which were previously fetched from `govuk_frontend_toolkit`. This PR:

- Adds the numbered steps icon as a black circle SVG within the CSS file, so the number can be inserted dynamically, rather than having separate .png files for each number.
- Adds a file download icon SVG
- Adds a warning SVG

## Why
These icons were originally fetched from govuk_frontend_toolkit. Because most of our frontend apps still have dependencies on govuk_frontend_toolkit, the icons are still working on the live site as the icons are still present. However, the more we remove toolkit from the apps, the more we will notice these icons stop working.

## Advisory
Screen Shot 2019-10-04 at 15 03 16
Screen Shot 2019-10-04 at 15 03 21

## Warning Callout
Screen Shot 2019-10-04 at 15 03 57
Screen Shot 2019-10-04 at 15 04 03

## Numbered Steps
Screen Shot 2019-10-04 at 15 03 39
Screen Shot 2019-10-04 at 15 03 43

## File Download
Screen Shot 2019-10-04 at 15 03 28
Screen Shot 2019-10-04 at 15 03 32

https://govuk-publishing-compo-pr-1154.herokuapp.com/component-guide/govspeak

### CODE REVIEW

**Comment by kr8n3r** [comment]
this fixes #1122

**Comment by sihugh** [comment]
❤️ the PR description

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## Unreleased
+
+* Add warning, file download and numbered steps icons that were originally fetched from govuk_frontend_toolkit ([PR #1154](https://github.com/alphagov/govuk_publishing_components/pull/1154))
```
Comment (andysellick) [inline CHANGELOG.md]: Nitpick: can title of PR match this?

**Comment by vanitabarrett** [comment]
@alex-ju @kr8n3r Are we happy with this being merged and removing legacy IE support? From what I can see, the icons fail to appear in IE8 and below, but I think the content still makes sense without these icons.

**Comment by alex-ju** [comment]
> Are we happy with this being merged and removing legacy IE support? From what I can see, the icons fail to appear in IE8 and below, but I think the content still makes sense without these icons.

Yes, I am. @kr8n3r what do you thin?

**Comment by kr8n3r** [comment]
yeah all good 👍

**Pull request merged**